src/component/componentAdmin/bodyAdmin/managerUsers/index.js

```diff
@@ -62,9 +64,18 @@ export default function ManagerUser() {
   const [mode, setMode] = useState("ADD");
   const [textSearch, setTextSearch] = useState("");
 
-  const toastAddSc = () => toast.success("add product success");
-  const toastDeleteSc = () => toast.success("delete product success");
-  const toaStEditSc = () => toast.success("edit product success");
+  const toastDeleteSc = () => toast.success("delete user success");
+
+  const toaStEditSc = () => toast.success("edit user success");
+
+  const toastSignupSc = () => toast.success("add user success");
+
+  const toaStSignUpEr = () => toast.warning("Email already used ");
```

Comment:
Chết dở, toàn không có i18n thế này

Reply:
tưởng trang admin không cần i18n nên ko thêm cái button đổi ngôn ngữ luôn,để bổ sung :)